src/main/java/seedu/address/logic/commands/CreateTagCommand.java

```diff
@@ -0,0 +1,71 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Adds a tag to the address book.
+ */
+public class CreateTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "create-tag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a tag to the address book. "
```

Comment:
Maybe "Creates Tag for the address book/Wedlinker"? 
"Adds a tag" seems to have the connotation of tagging to a contact

Reply:
Yup, I think that would be clearer.